### ISSUE TITLE
Abnormalities of the optic disc vasculature

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10941]
Preferred term label:
Abnormalities of the optic disc vasculature

Synonyms
Optic disc vascular abnormalities, Abnormal optic nerve head vasculature

Definition (free text, please give PubMed ID)
A condition characterized by structural or functional abnormalities in the blood vessels of the optic disc, including bleeding, tortuosity, dilation, shunting, or other unusual branching patterns. These abnormalities may be associated with conditions such as glaucoma, papilledema, leber optic neuropathy, or optic disc drusen, and can contribute to visual field defects or other optic neuropathies.

Parent term (use hpo.jax.org/app)
Abnormality of the optic disc (HP:12795)

Diseases characterized by this term
Glaucoma, Papilledema, Leber optic neuropathy

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01  Adding a "morphology" term right now. For the functional abn term -- it is sometimes unclear if we can really measure the pathophysiology at this level of granularity - if we can, let's discuss how to define and create a corresponding term. Thanks!

Adding new term: 
- ID: HP:6001168
- label: Abnormal optic disc vasculature morphology
- parents: [Abnormal optic disc morphology (HP:0012795)](https://hpo.jax.org/app/browse/term/HP:0012795)
- pmid: 
- def: A structural abnormality in the blood vessels of the optic disc, including bleeding, tortuosity, dilation, shunting, or other unusual branching patterns.
- comment: May be associated with conditions such as glaucoma, papilledema, leber optic neuropathy, or optic disc drusen, and can contribute to visual field defects or other optic neuropathies.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10941
- synonyms:
